extensions/poetry/PhylumExt.toml

```diff
@@ -4,5 +4,5 @@ entry_point = "main.ts"
 
 [permissions]
 run = ["poetry"]
-read = ["./poetry.lock", "./pyproject.toml"]
-write = ["./poetry.lock", "./pyproject.toml"]
+read = ["~/.cache/pypoetry", "~/.local/lib", "./"]
```

Comment:
Added `~/.local/lib` as a common path for unprivileged installations of Python. Other places should be covered by our whitelist. This doesn't take into account other exotic installation locations but I think it should be fine.

Reply:
Who commonly installs Python to `~/.local/lib`? Like which application.

Also I'm not opposed to adding `~/.local/lib` to our general exceptions. Very unlikely there's going to be sensitive data there.

Reply:
I'd probably think about adding the entirety of `~/.local`, honestly, as the chance that there's anything there that could pose more of a risk than anything in `/usr` does is slim.

What do you mean by "which application"? I do that myself because I usually stay on top of newer releases of Python, which aren't in package managers yet, plus I try to avoid privileged installations in `/usr/local`, and in those cases installation usually happens in `~/.local`.

Reply:
I definitely would **not** add all of `~/.local`. You can easily have sensitive data in there because it is used as a storage directory for applications. Say your text editor stores undo history there for example. So there's a **very high** chance *something* in there might be sensitive.

> What do you mean by "which application"? I do that myself because I usually stay on top of newer releases of Python, which aren't in package managers yet, plus I try to avoid privileged installations in /usr/local, and in those cases installation usually happens in ~/.local.

How did you decide on `~/.local/lib`? Not that I don't trust your judgement, but adding an exception just because one developer decided to put his stuff there isn't really a good reason. Was this documented somewhere as the recommended location?

Reply:
> I definitely would **not** add all of `~/.local`. You can easily have sensitive data in there because it is used as a storage directory for applications. Say your text editor stores undo history there for example. So there's a **very high** chance _something_ in there might be sensitive.

We could then add multiple exceptions, such as `~/.local/bin`, `etc`, `include` and so on. In the specific case of Python, no other paths are read.

> How did you decide on `~/.local/lib`? Not that I don't trust your judgement, but adding an exception just because one developer decided to put his stuff there isn't really a good reason. Was this documented somewhere as the recommended location?

I have been using `~/.local` as install prefix for as long as I remember the XDG spec being a thing. `lib` is just because that's where the Python installation is stored as a consequence. But I just checked and can't find an explicit reference to this, so we may want to let go of this assumption. Of course this creates a problem for users having installations in uncommon paths, but I'm not sure we want to support that just yet.